dockerfiles/load-tests/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM library/centos:centos7
+
+EXPOSE 8089
+
+RUN yum update --assumeyes \
+ && yum install --assumeyes \
+    git \
+    gcc \
+    make \
+    openssl-devel \
+    bzip2-devel \
+    libffi-devel
+
+WORKDIR /usr/src/
+RUN curl https://www.python.org/ftp/python/3.7.2/Python-3.7.2.tgz --output Python-3.7.2.tgz && tar xzf Python-3.7.2.tgz \
+ && cd Python-3.7.2 && ./configure --enable-optimizations && make altinstall && make clean \
+ && python3.7 -m pip install locustio locust websocket-client python-dateutil \
+
+WORKDIR /
+RUN git clone --single-branch --branch=383-create-locust-load-tests-for-OEM-testing https://github.com/ScrewTSW/rh-che.git rh-che-loadtesting
```

Comment:
Is it really supposed to clone branch from your repo? I think that all stuff should be in rh-che repo.

Reply:
good point. the tests are now in the master ;)